repository_service_tuf_worker/repository.py

```diff
@@ -396,6 +412,46 @@ def _send_publish_targets_task(self, task_id: str):  # pragma: no cover
             acks_late=True,
         )
 
+    def save_settings(self, root: Metadata[Root], settings: Dict[str, Any]):
+        """
+        Save settings to the repository settings.
+
+        Args:
+            root: Root metadata
+            settings: payload settings
+        """
+        logging.info("Saving settings")
+        for role in Roles:
+            rolename = role.value.upper()
+            threshold = 1
+            num_of_keys = 1
+            if rolename == Roles.ROOT.value.upper():
+                # The key to the root role is the name of the root file which
+                # uses consistent snapshot or in the format:
+                # <VERSION_NUMBER>.root.json
```

Comment:
I don't think that's true.
In our last iteration of bootstrap ceremony payload changes, we made the key to the `root` information to be just `root` and not the metadata file name anymore or using the version in any way.